pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -16,6 +16,12 @@ title: Troubleshooting
   * [Wifi doesn't connect on RPI3](#wifi-connection-problems-on-rpi3)
 * [NVIDIA Jetson TX2](#nvidia-jetson-tx2)
   * [Unable to flash a board that was previously running L4T 28.x](#unable-to-flash-a-board-that-was-previously-running-l4t-28x)
+  * [Are L4T 28.4 and 28.5 relases supported in BalenaOS](#are-L4T-28.4-and-28.5-releases-supported-in-BalenaOS)
+  * [Can a hostOS update from an older L4T 28.X based release to a newer L4T 32.Y be interrupted? Can I boot back into the old OS?](#can-a-hostOS-update-from-an-older-L4T-28.X-based-release-to-a-newer-L4T-32.Y-be-interrupted?-Can-I-boot-back-into-the-old-OS?)
+* [NVidia Jetson Nano SD-CARD and Jetson Nano 2GB](#nvidia-jetson-nano-sd-card-and-jetson-nano-2GB)
```

Comment:
The correct spelling as per their website and press releases is `NVIDIA`.

I think we are using it incorrectly across all of our products, but we could start by fixing it in this doc? Maybe we need more folks to weigh in.
```suggestion
* [NVIDIA Jetson Nano SD-CARD and Jetson Nano 2GB](#nvidia-jetson-nano-sd-card-and-jetson-nano-2GB)
```
There are 12 instances of `NVidia` on this page and one instance of `Nvidia` so we need to pick one and stick with it.

Reply:
@vipulgupta2048 @andrewnhem how do we standardize our spelling/capitalization of external brands?

Reply:
We use what they use, just like `Raspberry Pi` 
But, capital Nvidia seem weird IMHO

Reply:
@klutchell @vipulgupta2048 I think we should default to how they use it on their brand, NVIDIA. Exception being like in code. It'll look odd, but it beats being asked to do it later when we're big and famous and namedropping NVIDIA everywhere.